e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -831,7 +831,7 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
     cy.get("@metricId").then(metricId => {
       const questionDetails = {
         query: {
-          "source-table": `card__${metricId}`,
+          "source-table": ORDERS_ID,
```

Comment:
Just for my own education: why is this change needed? I realize it's fixing the failing e2e test, but why can't `card__${metricId}` be used here (other tests in this file still seem to use this pattern)?

Reply:
Oh, probably because removing source metrics is precisely what this PR / feature is about, and the other test using `card__${metricId}` is being skipped via `it.skip`? : )

Reply:
Well done! Exactly, this is what this feature is about, although in a normal world it should have been failing because of earlier changes in the QP (query processor).